Triangulation_2/include/CGAL/Triangulation_data_structure_2.h

```diff
@@ -312,7 +312,9 @@ class Triangulation_data_structure_2
 			  Face_handle f1, 
 			  Face_handle f2, 
 			  Face_handle f3);
+
```

Comment:
set_adjacency() must be public as the reconstruction algorithm will construct a TDS_2

Concerning orient_2 , it is about a small optimisation (replace find+insert   with insert, and using the returned bool)